components/authz-service/storage/errors_test.go

```diff
@@ -0,0 +1,19 @@
+package storage_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/chef/automate/components/authz-service/storage"
+)
+
+func TestNewProject(t *testing.T) {
```

Comment:
The test name is misleading, how about
```suggestion
func TestNewErrTxCommit(t *testing.T) {
```

Reply:
![](https://www.dictionary.com/e/wp-content/uploads/2018/03/copypasta.jpg)